app/models/comment.rb

```diff
@@ -293,10 +304,13 @@ def users_following_thread
   end
 
   def deliver_reply_notifications
+    notified = []
```

Comment:
Will this list ever be different from `users_following_thread`? If you use that list directly you won't have dupes, either.

Reply:
Yes, because not all people who follow the thread have `email_replies` or `pushover_replies` enabled.